test/unit/action/wallet.spec.js

```diff
@@ -221,6 +221,23 @@ describe('Action Wallet Unit Tests', () => {
     });
   });
 
+  describe('initInitialDeposit()', () => {
+    it('should navigate to new address screen if address is non-null', () => {
+      store.walletAddress = 'non-null-addr';
+      wallet.initInitialDeposit();
+      expect(nav.goNewAddress, 'was called once');
+      expect(nav.goWait, 'was not called');
+    });
+
+    it('should stay on wait screen until address is non-null', async () => {
+      store.walletAddress = null;
+      wallet.initInitialDeposit();
```

Comment:
Add check that `expect(nav.goNewAddress, 'was not called');` was not called before setting the store value.

Reply:
fixed